rero/rero-ils-ui
# routing: redirect after saving a resource

* Adds new routing type for items to redirect after save.
* Corrects the processing of the record according (create vs update).
* Closes rero/rero-ils/#376

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>
Co-Authored-by: Sébastien Délèze <sebastien.deleze@rero.ch>
Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

## Why are you opening this PR?

This task was done for US985, but this branch has not been merged at the end of the sprint. Meanwhile, the routing has been refactored, and so needed the code related to this task.

## How to test?

Create or edit a holding item. You should be redirected to the document detailed view.

## Code review check list
